packages/tizen_bundle/README.md

```diff
@@ -0,0 +1,133 @@
+# tizen_bundle
+
+Tizen bundle APIs.
+
+## Usage
+
+To use this package, add `tizen_bundle` as a dependency in your `pubspec.yaml` file.
+
+```yaml
+depenedencies:
+  tizen_bundle:
+
+```
+
+## Adding Content to a Bundle
+
+The bundle content is in the from of key-value pairs, The key is always a string. The value can be of the following types:
+
+**Table: Bundle value types**
+| Value constant           | Value type        |
+|--------------------------|-------------------|
+| `BundleType.string`      | String (default)  |
+| `BundleType.strings`     | String list       |
+| `BundleType.bytes`       | Bytes             |
```

Comment:
To clarify the meaning, how about

- Value constant → Type enum
- Value type → Dart type
- String list → List\<String>
- Bytes → Uint8List

Also why is String marked as "default"?

Reply:
Done. 
'default' means string is recomended. I've removed it. I thinks it's not needed.